docs/contribute/documentation.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Add a topic
+parent: How to contribute
+nav_order: 1
+---
+
+# Add or change a topic in the Knowledge Base
+
+Each documentation file has the same setup:
+
+- First a set of variables like the title and the parent.
+- Then the content.
+
+The documentation files live in the directory `/docs/` and are named `[title].md`.
+For example `contribute.md`. Keep the name short and unique. Use `-` instead of spaces in the filename.
+
+Each directory has an `index.md` which is the file that directly links to the directory topic. For example: 
+`/docs/about/index.md` displays [About this website](/docs/about/).
+Other files for this topic are also placed in this directory and assigned a 'parent'.
+
+## The variables at the top of the file for a topic
+
+Example for `/docs/about/index.md`:
+```
+---
+layout: default
+title: Accessibility statement
+parent: About this website
+description: "Accessibility statement of the website wpaccessibility.org."
+nav_order: 2
+---
+```
+Use for the `parent` the `title` of the parent and not the filename.
+
+## The content of the post
+
+The page [Content kitchen sink](/docs/contribute/content-kitchen-sink/) describes and displays all options for the content, like: markdown, images, videos, code, callouts and how to indicate a different langauge in the text.
+
+## How to start
+
+If you have an idea for a blog post, please open an [issue on GitHub](https://github.com/wpaccessibility/wp-a11y-docs/issues) and share your idea first.
```

Comment:
This is the same copy from `articles.md`; probably needs to change for this context.